Lab 1/Lab1.ipynb

```diff
@@ -2,11 +2,539 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(0,10,11)\n",
+    "pmf2 = stats.binom.pmf(x,10,0.3)\n",
+    "\n",
+    "plt.plot(x,pmf2)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ea73d79-b9c9-48c0-944d-c508ed70342c",
+   "metadata": {},
+   "source": [
+    "When the probability is away from 0.5, however, we begin to see a skew in the distribution. The peak of the distribution nears the expected value of successful events."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d038306-74e3-43ee-801f-cb34d5a57bce",
+   "metadata": {},
+   "source": [
+    "#### 3C.\n",
+    "\n",
+    "Let's propose a scientific question.\n",
+    "\n",
+    "Using our graph above, with n = 10 and p = 0.3:\n",
+    "\n",
+    "__What is the probability of X being successful for 6 trials or more?__ \n",
+    "\n",
+    "We can do what we did similar to calculating the cumulative distribution function (and subtracting from 1 depending on the question you're asking). We just need to make sure to add up all the probabilities of all values within our range, which is 6-10 inclusive."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "id": "734eaaba-0a4e-4e74-b593-c26d136cf02c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.04734898740000004 is the probability of 6 or more successful events of X.\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob2 = (1 - stats.binom.cdf(k = 5, n = 10, p = 0.3))\n",
+    "print(str(prob2) + ' is the probability of 6 or more successful events of X.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "033458f9-16f5-46cf-a3c4-8bac458504e7",
+   "metadata": {},
+   "source": [
+    "Here we calculated the cumulative distribution function up to and including 5 successes. To get 6 and above, we simply subtract this value from 1."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8949e8ce-1e99-47f7-bbc5-a45b2a280f55",
+   "metadata": {},
+   "source": [
+    "#### 3D."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f7bd076-e1d5-421a-9685-cfd357c4d9d7",
+   "metadata": {},
+   "source": [
+    "Probabilities and sigmas are also discrete in these non-continuous distributions. So compared to continuous probability distributions, instead of thinking in terms of integrals and infinitesimals, we must think of actually summing the probabilities of these discrete values. The biggest catch might be, unlike continuous distributions, we can also calculate the probability at a single value of k."
```

Comment:
I really like the point you made in your last sentence; I'm going to have to add that to mine :P